share/picongpu/tests/Pusher/include/picongpu/param/particle.param

```diff
@@ -0,0 +1,114 @@
+/* Copyright 2013-2022 Axel Huebl, Rene Widera, Benjamin Worpitz,
+ *                     Richard Pausch, Klaus Steiniger
```

Comment:
might your name be missing here @MRZirper

Reply:
I don't know. And it is already very crowded there...